custom-recipes/ml-assisted-labeling-deephub-export/recipe.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+import json
+import logging
+import dataiku
+from dataiku.customrecipe import get_recipe_config, get_input_names_for_role, get_output_names_for_role
+
+
+def format_labeling_plugin_annotations(image_annotations):
+    """ Format labeling annotations to be compatible with dss object detection (deephub)"""
+
+    deephub_image_annotations = []
+    # if images were skipped during labelling plugin, their annotations is nan:
+    image_annotations = json.loads(image_annotations) if isinstance(image_annotations, str) else []
+    for annotation in image_annotations:
+        deephub_image_annotations.append(
+            {"bbox": [annotation.get("left"), annotation.get("top"), annotation.get("width"), annotation.get("height")],
+             "area": annotation.get("height") * annotation.get("width"),
+             "iscrowd": False,
+             "category": annotation.get("label")
+             })
+    image_annotations = json.dumps(deephub_image_annotations)
+    return image_annotations
+
+
+input_dataset = dataiku.Dataset(get_input_names_for_role("input_dataset")[0])
+
+# copy all the columns from input dataset (only target one will be changed)
+output_df = input_dataset.get_dataframe()
+
+target_column = get_recipe_config()["target_column"]
+if target_column not in output_df.columns:
+    raise Exception("Column {} not in input_df with columns [{}]".format(target_column, list(output_df.columns)))
```

Comment:
Shouldn't that be `input_df` to match the error?
```suggestion
if target_column not in input_df.columns:
    raise Exception("Column {} not in input_df with columns [{}]".format(target_column, input_df.columns.tolist()))
```

Or just change the error message if we really want to check `output_df`

Reply:
well at this point theu are the same as we copy input into output just before, but you're right that's not clear i'll clean this part